examples/basics/visuals/graph.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+"""
+This example demonstrates how to visualise a NetworkX graph using the
+GraphVisual.
+"""
+
+import sys
+
+import networkx as nx
+
+from vispy import app, gloo, visuals
+from vispy.visuals.graphs import layouts
+from vispy.visuals.transforms import STTransform
+
+
+class Canvas(app.Canvas):
+    def __init__(self):
+        app.Canvas.__init__(self, title="Simple NetworkX Graph",
+                            keys="interactive", size=(600, 600))
+
+        self.graph = nx.fast_gnp_random_graph(100, 0.02)
+
+        self.visual = visuals.GraphVisual(
+            nx.adjacency_matrix(self.graph),
+            layout=layouts.get('force_directed'),
```

Comment:
Ah it's changed here locally but I didn't commit those changes because I experiment often a bit with the number of nodes and probability of the edge